SketchUp/Geom/BoundingBox.rb

```diff
@@ -253,10 +249,10 @@ def initialize
   #   boundingboxes actually overlap - then call this to get the resulting
   #   boundingbox.
   #
-  # @param boundingbox
+  # @param boundingbox [Geom::BoundingBox]
```

Comment:
We've been formatting tags to have the type before the parameter name.

Reply:
Just for the record, in the YARD docs itself, `@param [types] name description` is the OLD way, and what I originally proposed (in the Guide) is what the YARD doc encourage, ie: `@param name [types] description`.

I'm weary of having to go round and round on these same stupid issues, So I'm leaving this project and deleting my repo copy. See ya'